com.danielerolli.beavernotes.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: com.danielerolli.beavernotes
```

Comment:
```
Host danielerolli.com not found: 3(NXDOMAIN)
```

This app-id doesn't meet the requirements. Even though you acknowledged

> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

Reply:
Yeah, I forgot to change it, and I've just pushed a commit to fix it, making it com.beavernotes.beavernotes.